src/v0/sources/shopify/util.js

```diff
@@ -158,6 +194,7 @@ const updateCartItemsInRedis = async (cartToken, newCartItemsHash, metricMetadat
   try {
     await RedisDB.setVal(`${cartToken}`, value);
   } catch (e) {
+    logger.debug(`{{SHOPIFY::}} itemsHash set call Failed due redis error ${e}`);
```

Comment:
`logger.debug(`{{SHOPIFY::}} Failed to set itemsHash due to a Redis error: ${e}`);`